app/Http/Controllers/User/HomeController.php

```diff
@@ -16,6 +16,7 @@ public function index(Request $request)
     {
         $request = $request->all();
         $teachers = Teacher::orderBy('id','ASC');
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Generic.Functions.FunctionCallArgumentSpacing.NoSpaceAfterComma</br>**Severity**: error</br>**File**: app/Http/Controllers/User/HomeController.php</br>**Line**: 18</br>No space found after comma in function call